### PR TITLE
Added installation guide to online documentation.

### PR DESCRIPTION
**Pull request by kroczi**

### CODE REVIEW

File: docs/source/gi_gui.rst [hunk 1]
```diff
@@ -1,4 +1,96 @@
...
+
+Parameters
+..........
+Comparison parameters can be provided in two ways: by manually setting up parameters or using already saved configurations (from previous comparisons if they were saved).
+
+If you choose manual way you have to handle following parameters:
+ - *Dose Difference* - normalization factor for Dose Difference metric. In percentages. **It is mandatory.**
+ - *Distance To Agreement* - normalization factor for Distance To Agreement metric. In millimeters. **It is mandatory.**
+ - *Algorithm version* - determines whether a Dose Difference's normalization factor should be multiplied by Reference Value (*Global*) or by current value at reference image (*Local*)
+ - *Reference Value* - together wit Dose Difference parameter creates normalization factor for Dose Difference component in global version of algorithm.
```
Comment (patrykks) [inline docs/source/gi_gui.rst]: wit -> with

File: docs/source/gi_gui.rst [hunk 2]
```diff
@@ -81,7 +81,7 @@ The *yAGIT* application provides also a feature of tooltips - small explanation
 
 Application logs
 ,,,,,,,,,,,,,,,,
-
+The *yAGIT* application is equipped with logging mechanism. It is turned off by default. In order to turn it on, one can go to *Options* located in menu bar on the top of the application and select "Enable Logging". The logs will be save in logs.log files located in *gi_gui/* directory. It can be very useful in tracking application flow.
```
Comment (kroczi) [inline docs/source/gi_gui.rst]: "Enable Logging" -> *Enable Logging*
logs.log -> *logs.txt*

File: docs/source/gi_gui.rst [hunk 3]
```diff
@@ -94,3 +94,23 @@ Gamma Index calculation
 4. Set parameters manually (including mandatory: Dose Difference, Distance To Agreement, Algorithm Version, Initial Gamma Index Value, Dimension) or load them from configuration using *Manage Parameters* button
 5. Use *Calculate Gamma Index* button to start calculation
 
+Saving result image and statistics into files
+,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
+1. In the application make at least one comparison
```
Comment (kroczi) [inline docs/source/gi_gui.rst]: In the application?
make -> perform

File: docs/source/spiral_version.rst [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+
+Because of this, between |ref| point and the center of the spiral may emerge a gap, we called this distance *closest*. While comapring points from spiral with current |ref| point, there may occour a situation, when *dta* component will exceed the Gamma Index value, and according to the above assumptions we should end the computations for given |ref| point. But because of the gap, there may come out that the next |tar| point on the spiral, despite being located further or as close as the previous one form the center of the spiral, lies a bit closer to the |ref| point. In the face of this fact we had no choice but to weaken our termination condition. While checking the stopping criterion we calculate *dta* based on the distance of current |tar| point form the center of the spiral reduced by the value of *closest*. This gives us the certainty that if given point fails the termination condition, then all the next points will not have the *dta* component smaller than the already found Gamma Index value.
```
Comment (patrykks) [inline docs/source/spiral_version.rst]: While comapring?

Comment (patrykks) [inline docs/source/spiral_version.rst]: occour?

File: docs/source/spiral_version.rst [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+
+Summary
+.......
+
+Thanks to the modifications described above, we perform only a few comparison per a reference point and we do not need to run through the whole target image for each reference point.
+
+The classic version for images of high resolution or 3-dimensional ones required a huge amount of time (counted in minutes ore even hours) to perform a comparison. But our spiral version it takes only about a minute, and often, in case of quite simmilar images, the computation lasts only a few seconds.
```
Comment (patrykks) [inline docs/source/spiral_version.rst]: ore

Comment (patrykks) [inline docs/source/spiral_version.rst]: "But our spiral version it takes only" is it required here ?

File: docs/source/gi_core_2d_example_client.rst [hunk 6]
```diff
@@ -38,8 +38,8 @@ They are used during 2.5D or 2D for 3D comparison:
 First of all it is required to obtain reference and target images for the comparison.
 For this example there are prepared sample text files containing such images and *read2DMatrixFromTxt()* function that parses the image files and returns them as an linearized array with image spatial parameters.
 These parameters are: coordinates of top left corner (start), pixel spacing and  number of pixels along each dimensions (resolution).
-The image samples can be found under *gi_clients/images* in the github repository.
-
+The image samples can be found under *gi_clients/images* in the repository with code.
```
Comment (kroczi) [inline docs/source/gi_core_2d_example_client.rst]: I would generally removed the part " in the repository with code". It is not important and also does not sound well.

File: docs/source/gi_core_2d_example_client.rst [hunk 7]
```diff
@@ -38,8 +38,8 @@ They are used during 2.5D or 2D for 3D comparison:
...
+0
```
Comment (kroczi) [inline docs/source/gi_core_2d_example_client.rst]: what is this for?

File: docs/source/gi_gui.rst [hunk 8]
```diff
@@ -91,7 +91,7 @@ Gamma Index calculation
 1. Open application from *gi_gui/* directory
 2. Choose reference image using *Set Reference File* button
 3. Choose target image using *Set Target File* button
-4. Set parameters manually (including mandatory: Dose Difference, Distance To Agreement, Algorithm Version, Initial Gamma Index Value, Dimension) or load them from configuration using *Manage Parameters* button
+4. Set parameters manually (including mandatory: Dose Difference, Distance To Agreement, Algorithm Version, Initial Gamma Index Value, Dimension (suitable for selected DICOMS)) or load them from configuration using *Manage Parameters* button
```
Comment (kroczi) [inline docs/source/gi_gui.rst]: DICOMS -> DICOMs

File: docs/source/introduction.rst [hunk 9]
```diff
@@ -43,4 +43,5 @@ Limitations
  - lack of built-in visualization tool
  - support only for files of DICOM type (although user can easily write their own wrapper and integrate it with our application)
  - lack of support for images not specified on a regular grid (images with different pixel spacing along one axis)
+ - DICOMDIR images are not supported
```
Comment (kroczi) [inline docs/source/introduction.rst]: please adapt to the convention

File: docs/source/conf.py [hunk 10]
```diff
@@ -84,7 +84,7 @@
 # built documents.
 #
 # The short X.Y version.
-version = u'0.0.1'
+version = u'0.1.0'
 # The full version, including alpha/beta/rc tags.
 release = u'0.0.1'
```
Comment (kroczi) [inline docs/source/conf.py]: what is the difference between version and release?

File: docs/source/gammma_index.rst [hunk 11]
```diff
@@ -3,15 +3,18 @@ Gamma Index
 
 1. Gamma Index - what it is
 ---------------------------
-The Gamma Index is a tool for dose distribution comparison. It combines both dose difference (DD) and distance to agreement (DTA) into a single quantity. Though it is an effective measure, making up for the inadequacy of DD or DTA alone, its calculation can be very time-consuming. In a classic version for a k-D space with N quantization levels in each dimension, the complexity of the exhaustive search is O(N^2k).
+The Gamma Index is a way of comparing two images called reference and target. It has gain lot of popularity in radiotherapy in comparing two images of dose distribution.
+It combines two other methods of comparing images: dose difference (DD) and distance to agreement (DTA). Both of them will be explained in details int the following section. This procedure enabled managing situations in which source comparison way, wasn't doing so well while still using it's advantages.
```
Comment (patrykks) [inline docs/source/gammma_index.rst]: int

**Pull request merged**